app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -26,7 +26,17 @@ angular.module("openshiftConsole")
         }];
         scope.claim.selectedLabels = [];
         DataService.list({group: 'storage.k8s.io', resource: 'storageclasses'}, {}, function(storageClasses) {
-          scope.storageClasses = storageClasses.by('metadata.name');
+          var noclass = {
+            metadata: {
+              name: "No Storage Class",
+              labels: {},
+              annotations: {
+                description: "No storage class will be assigned unless a default class has been assigned by the system administrator."
+                }
+            }
+          };
+          storageClasses._data["No Storage Class"] = noclass;
```

Comment:
It's also cached by DataService and shared across controllers, so anyone else asking for storage classes would get the "no storage class" item you added.